api-module-library/microsoft-teams/test/graph-user.test.js

```diff
@@ -25,66 +24,153 @@ describe(`${config.label} API Tests`, () => {
+            userId = user.id;
         });
         it('Should retrieve information about the Organization', async () => {
             const org = await api.getOrganization();
-            org.should.exist;
+            expect(org).toBeDefined();
+            tenantId = org.id;
         });
     });
 
+
+    let teamId;
+    it('Get joined teams', async ()=> {
+        api.setTenantId(tenantId);
+        const joinedTeams = await api.getJoinedTeams();
+        expect(joinedTeams).toHaveProperty('value');
+        teamId = joinedTeams.value.slice(-1)[0].id;
+    });
+
+
+
     let createChannelResponse;
-    describe('Create Channel Request', () => {
+    // skip channel creation tests to avoid private channel limitations
+    // unskip at any time to test Channel creation behavior
+    describe.skip('Channel Requests', () => {
         it('Should create channel', async () => {
+            api.setTeamId(teamId);
             const body = {
                 "displayName": `Test channel ${Date.now()}`,
                 "description": "Test channel created by api.test",
                 "membershipType": "private"
             }
             createChannelResponse = await api.createChannel(body);
-            createChannelResponse.should.exist;
+            expect(createChannelResponse).toBeDefined();
         });
-    });
-
-    const mwebberUserId = 'c1cb384d-8a26-464e-8fe3-7117e5fd7918'
-    describe('Add user to channel Request', () => {
-        it('Should create channel', async () => {
+        it('Should add user to channel', async () => {
             const conversationMember = {
                 '@odata.type': '#microsoft.graph.aadUserConversationMember',
                 roles: [],
-                'user@odata.bind': `https://graph.microsoft.com/v1.0/users(\'${mwebberUserId}\')`
+                'user@odata.bind': `https://graph.microsoft.com/v1.0/users(\'${userId}\')`
             };
             const response = await api.addUserToChannel(createChannelResponse.id, conversationMember);
-            response.should.exist;
+            expect(response).toBeDefined();
+        });
+        it('List users in channel Request', async () => {
+            const response = await api.listChannelMembers(createChannelResponse.id);
+            expect(response).toBeDefined();
+            expect(response.value[0].userId).toBe(userId)
+        });
+        it('Delete the created channel', async () => {
+            const response = await api.deleteChannel(createChannelResponse.id);
+            expect(response.status).toBe(204);
         });
     });
 
-    describe('List users in channel Request', () => {
-        it('Should create channel', async () => {
-            const response = await api.listChannelMembers(createChannelResponse.id);
-            response.should.exist;
-            expect(response.value[0].userId).toBe(mwebberUserId)
+    describe('User App requests', ()=> {
+        const externalAppId = 'd0f523b9-97e8-42d9-9e0a-d82da5ec3ed1'
```

Comment:
Should this be a test ENV, or is this a random app off the App Store?

Reply:
right now this is our test app, although I was thinking of switching it to another app specifically just for testing, one off the store could work